src/components/SpaceAvatar.vue

```diff
@@ -1,13 +1,12 @@
-<script setup>
+<script setup lang="ts">
 import { computed } from 'vue';
 import { formatBytes32String } from '@ethersproject/strings';
-import { getUrl } from '@snapshot-labs/snapshot.js/src/utils.ts';
+import { getUrl } from '@snapshot-labs/snapshot.js/src/utils';
 
-const props = defineProps({
-  space: Object,
-  size: String,
-  symbolIndex: [String, Number]
-});
+const props = defineProps<{
+  space: Record<string, any>;
+  size: string;
+}>();
```

Comment:
why remove symbolIndex?

Reply:
Because it's not being used. Or what am I missing?